Listing description update (#99)
Fabulously-Optimized/wiki: listing-description/en-US.md

A Fabric-based Minecraft modpack focusing on performance and graphics enhancements.

> [![Video review](https://img.youtube.com/vi/bb8G9X5Q_4I/hqdefault.jpg)](https://www.youtube.com/watch?v=bb8G9X5Q_4I)

Reviewer: On modrinth and on the CF description, this should likely be replaced by

```html
<iframe allowfullscreen="allowfullscreen" src="https://www.youtube.com/embed/bb8G9X5Q_4I" height="358" width="638">
</iframe>
```

Author: Nah. It used to be an embed before, but an embed is not scalable (in the listings) for mobile and even previewing it wastes computer resources, hence thumbnails were chosen. Since the upcoming website will display one language at once and is better scalable, there an actual embed could be used.

### Performance

> Fabulously Optimized is faster than OptiFine thanks to [modern performance mods][1] such as Sodium, Lithium and others! The bundled mods have been carefully selected to be safe to your worlds and bring noticeable speed benefits on any device.

Reviewer: ```suggestion
Fabulously Optimized is faster than OptiFine, thanks to [modern performance mods][1] like Sodium, Lithium, and others. The bundled mods have been thoughtfully selected to ensure safety in your worlds and provide noticeable speed benefits on any device.
```

> Fabulously Optimized is faster than OptiFine thanks to [modern performance mods][1] such as Sodium, Lithium and others! The bundled mods have been carefully selected to be safe to your worlds and bring noticeable speed benefits on any device.

Reviewer 2: "ensure safety in" is a bit excessive

> Fabulously Optimized is faster than OptiFine thanks to [modern performance mods][1] such as Sodium, Lithium and others! The bundled mods have been carefully selected to be safe to your worlds and bring noticeable speed benefits on any device.

Author: I'm not used to having a comma before "and" but I guess it is valid English indeed (the Oxford comma). The mods do not "ensure safety" like an antivirus, they just don't mess it up.
Partly implemented

### OptiFine parity

> Fabulously Optimized [supports most OptiFine features][2], including shaders, connected textures and zoom. If your resource pack says "OptiFine required", it will likely work on here as well. [You can also get a free cape!][3]

Reviewer: ```suggestion
Fabulously Optimized [offers support for most OptiFine features][2], including shaders, connected textures, and zoom. If your resource pack states "OptiFine required," it will probably function here as well. [Additionally, you can get your own free cape.][3]!
```

> Fabulously Optimized [supports most OptiFine features][2], including shaders, connected textures and zoom. If your resource pack says "OptiFine required", it will likely work on here as well. [You can also get a free cape!][3]

Author: I don't "offer support" like providing help or having add-ons, I already "have the support". Also, reverted the cape sentence.

### Feels familiar

> Fabulously Optimized [includes features that let you play Minecraft just as you remember it][4], no matter what version you've used before. That means you can use game controllers, play local worlds with friends, see advancements in full screen and more. 

Reviewer 2: Personally, I'll prefer a hybrid like:
`Fabulously Optimized [includes some minor features][4] that lets you play Minecraft just as you remember it, without breaking the vanilla feel. For example, you can use game controllers, play local worlds with friends, see advancements in full screen, and more.`

Reviewer: ```suggestion
Fabulously Optimized [incorporates features that allow you to experience Minecraft just as you remember it][4], regardless of the version you've previously used. This means you can use game controllers, play local worlds with friends, view advancements in full-screen mode, and much more.
```

### Works everywhere

> Fabulously Optimized has no content mods, so you can play on any world or server. While you can play the pack with any game launcher, [we encourage and support these as they give you the best experience.][6]

Reviewer 2: play is repeated, so i'il replace the second with something like `use` or `run`. 
`[we encourage and support these as they give you the best experience.]` => `[we recommend these for the best experience possible.]`

Reviewer: ```suggestion
Fabulously Optimized doesn't include content mods, enabling you to play on any world or server. While you can use any game launcher to play the pack, [we recommend and provide support for specific launchers that offer the best experience][6].
```

### Up to date

> Thanks to Fabric mod loader, Fabulously Optimized [receives fast updates][5] and is one of the first modpacks to upgrade to newer Minecraft versions. [Updating the modpack is easy][17] and will preserve your configuration changes, where possible.

Reviewer 2: I would emphasize more the fact that fo is available soon after the release, faster than optifine. Also with these changes the "Thanks to fabric loader" doesn't make much sense here.

I think a new title like this fits well: 
`### Built on Fabric

Fabric is a lightweight, fast, and up-to-date mod loader that has huge mod support.
You can use Fabulously Optimized for modded Minecraft gameplay (and as a base for your own modpack, thanks to his open-source license).
`
the between-parenthesis part is optional.

Reviewer: ```suggestion
Thanks to the Fabric mod loader, Fabulously Optimized [receives swift updates][5] and stands as one of the first modpacks to upgrade to newer Minecraft versions. [Updating the modpack is a straightforward process][17] and will preserve your configuration changes, where possible.
```

Author: Re-added the separate Fabric paragraph and rewrote both.

### Open development

> Fabulously Optimized [is open-source][8] and so are most of its mods. We welcome suggestions from the users and collaborate with mod developers to create a better user experience for everyone.

Reviewer: ```suggestion
Fabulously Optimized [is open-source][8], and so are most of its mods. We welcome user suggestions and collaborate with mod developers to enhance the user experience for everyone.
```

Author: Partly implemented

### Helpful community

> We have a growing community of friendly members who like to discuss the pack, [translate its content][7] and help each other. [Join us today!][10]

Reviewer: ```suggestion
We have a growing community of friendly members who enjoy discussing the pack, [translating its content][7], and assisting each other. [Join us today][10]!
```
